impeller/docs/babys_first_triangle.md

```diff
@@ -162,7 +162,7 @@ And in the body, set the color of the fragment to the this input.
 frag_color = v_color;
 ```
 
-Notice we didn't do anything to perform the color mixing. That's because the rasterizer interpolates the values between stages. Since the varies depending on the pixel, we call these "varyings" and use the `v_` prefix for such variables.
+We didn't do anything to perform the color mixing. That's because the rasterizer interpolates the values between stages. Since the varies depending on the pixel, we call these "varyings" and use the `v_` prefix for such variables.
```

Comment:
trivial nit, feel free to ignore

Maybe "We didn't do anything to perform the color mixing because the rasterizer interpolates..."?

Whichever, either's fine.